sync/class.jetpack-sync-module-comments.php

```diff
@@ -40,6 +43,23 @@ public function init_listeners( $callable ) {
 		$this->init_meta_whitelist_handler( 'comment', array( $this, 'filter_meta' ) );
 	}
 
+	/**
+	 * @param array $data       The new, processed comment data.
+	 * @param array $comment    The old, unslashed comment data.
+	 */
+	public function handle_comment_contents_modification( $data, $comment ) {
+
+		if ( $data['comment_content'] != $comment['comment_content'] ) {
```

Comment:
I think we should compare more then just the comment content, in the ui you can change the name, email and the website as well.